photon-core/src/main/java/org/photonvision/common/dataflow/networktables/NetworkTablesManager.java

```diff
@@ -164,6 +165,9 @@ public void setConfig(NetworkConfig config) {
         } else {
             setClientMode(config.ntServerAddress);
         }
+
+        VisionModuleManager.getInstance().setShouldPublishProto(config.shouldPublishProto);
```

Comment:
Why not have NtDataPublisher pull directly from ConfigManager.getInstance().getConfig().getNetworkConfig().shouldPublishProto? Why this misdirection?

Reply:
Uhhhh, me no do photon-core. I didn't know that was a thing.

Reply:
In general, pipelines pulling settings from global networking settings isn't really a thing we commonly do. Usually that's all held per-pipeline. But making this a per-pipeline setting seems like also a bad idea. So I think my proposal makes more sense